runtime/src/main/java/org/corfudb/runtime/collections/streaming/LRStreamingTask.java

```diff
@@ -0,0 +1,56 @@
+package org.corfudb.runtime.collections.streaming;
+
+import com.google.protobuf.Message;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.StreamListener;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableSchema;
+import org.corfudb.runtime.view.TableRegistry;
+import javax.annotation.Nonnull;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+
+public class LRStreamingTask<K extends Message, V extends Message, M extends Message> extends StreamingTask {
+
+    public static final UUID LR_MULTI_NAMESPACE_STREAM_ID = UUID.randomUUID();
+
+    public LRStreamingTask(CorfuRuntime runtime, ExecutorService workerPool,
+                           @Nonnull Map<String, String> nsToStreamTags, @Nonnull Map<String, List<String>> nsToTables,
```

Comment:
nit -> nsToStreamTag (singular) as there can only be one stream tag per namespace

Reply:
nit: nsToTables -> seems more like nsToTableNames

Reply:
We should add a precondition check where same namespaces in nsToStreamTags is present in nsToTables